tools/astarte_dev_tool/lib/constants/system.ex

```diff
@@ -0,0 +1,31 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule AstarteDevTool.Constants.System do
+  @command "docker"
```

Comment:
This was recently added to the Elixir docs, tl;dr: if you don't have some very specific needs you should directly make your functions return the constant instead of assigning that to a module attribute and then returning the module attribute
https://hexdocs.pm/elixir/module-attributes.html#as-compile-time-constants

Reply:
I've just discussed with @eddbbt about it